homeassistant/components/suez_water/sensor.py

```diff
@@ -23,39 +27,37 @@
 })
 
 
-def setup_platform(hass, config, add_devices, discovery_info=None):
+def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the sensor platform."""
-    from pysuez.client import PySuezError
-    from pysuez import SuezClient
-
     username = config[CONF_USERNAME]
     password = config[CONF_PASSWORD]
     counter_id = config[CONF_COUNTER_ID]
     try:
         client = SuezClient(
             username, password, counter_id)
 
-        if client.check_credentials():
-            add_devices([SuezSensor(client)], True)
-        else:
+        if not client.check_credentials():
             _LOGGER.warning("Wrong username and/or password")
+        else:
+            add_entities([SuezSensor(client)], True)
 
     except PySuezError:
         _LOGGER.warning("Unable to create Suez Client")
-        return False
+        return
 
 
 class SuezSensor(Entity):
     """Representation of a Sensor."""
 
     def __init__(self, client):
         """Initialize the data object."""
-        self._name = "Suez Water Client"
+        self._name = COMPONENT_NAME
```

Comment:
Return the constant directly in the `name` property. We don't need the private instance attribute.